plugins/PrivacyManager/vue/src/ManageGdpr/ManageGdpr.vue

```diff
@@ -106,7 +107,7 @@
             <th>{{ translate('General_VisitorIP') }}</th>
             <th>{{ translate('General_UserId') }}</th>
             <th>{{ translate('General_Details') }}</th>
-            <th v-show="profileEnabled">{{ translate('General_Action') }}</th>
+            <th v-show="hasActions">{{ translate('General_Action') }}</th>
```

Comment:
The action will now be shown as soon as one record provides a `visitorId`, that way the link will be shown even if the visitlsprofile is disabled for the current site.